server/public/styles.css

```diff
@@ -68,3 +68,12 @@
   background-position: center;
   background-repeat: no-repeat;
 }
+
+.pedal-delete-button {
+  margin: 10px 0 !important;
```

Comment:
Are you sure you need all of these `!important`?

Reply:
The semantic UI-react components wont let me style certain things outside of the actual component.  Was not sure if using !important was the best way.